sdk/core/azure-core/src/main/java/com/azure/core/http/ContentType.java

```diff
@@ -12,6 +12,11 @@
      */
     public static final String APPLICATION_JSON = "application/json";
 
+    /**
+     * the default JSON Content-Type header.
```

Comment:
This should state XML right? XML isn't JSON :)

Reply:
Also lets uppercase `The`, if you could update all instances in this file that would be great!